tests/plugins/inlinetoolbar/cssloading.js

```diff
@@ -0,0 +1,17 @@
+/* bender-tags: inlinetoolbar */
+/* bender-ckeditor-plugins: inlinetoolbar,button,richcombo, wysiwygarea */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+	};
+	CKEDITOR.skinName = 'kama,/apps/ckeditor/skins/kama/';
+	var spy = sinon.spy( CKEDITOR.dom.document.prototype, 'appendStyleSheet' );
+	bender.test( {
+		'test loading css with path': function() {
+			assert.areSame( 'kama,/apps/ckeditor/skins/kama/', CKEDITOR.skinName, 'config skinName should be with path' );
+			sinon.assert.calledWith( spy, 'http://127.0.0.1:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css' );
```

Comment:
This test case fails for me.

```
Unexpected error: expected appendStyleSheet to be called with arguments http://127.0.0.1:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css
    appendStyleSheet(/apps/ckeditor/skins/kama/editor.css) at loadCss (http://tests.ckeditor.test:1030/apps/ckeditor/core/skin.js:168:22)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/skins/default.css) at Object.beforeInit (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/plugin.js:530:23)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css) at Object.beforeInit (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/plugin.js:531:23)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/skins/kama,/apps/ckeditor/skins/kama//balloonpanel.css) at Object.init (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/plugin.js:20:23)
Expected: undefined (undefined)
Actual:   undefined (undefined)
```

Our standard test domain is test `tests.ckeditor.test` but you should pick it dynamically.